src/abstractsparse.jl

```diff
@@ -16,6 +16,24 @@ Supertype for one-dimensional sparse arrays (or array-like types) with elements
 of type `Tv` and index type `Ti`. Alias for `AbstractSparseArray{Tv,Ti,1}`.
 """
 const AbstractSparseVector{Tv,Ti} = AbstractSparseArray{Tv,Ti,1}
+
+"""
+    AbstractSparseVectorC{Tv,Ti}
+
+Supertype for vectors stored using a compressed map.
+"""
+abstract type AbstractSparseVectorC{Tv,Ti} <: AbstractSparseVector{Tv,Ti} end
```

Comment:
Can this be `AbstractCompressedVector`? `C` is confusing to me every time I see it.

Reply:
sure